packages/presentation/lib/screen/home/home_page.dart

```diff
@@ -0,0 +1,171 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:presentation/screen/home/bloc/home_bloc.dart';
+import 'package:presentation/screen/home/show_dialog_widget.dart';
+
+class MyApp extends StatelessWidget {
+  const MyApp({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return BlocProvider(
+      create: (context) => HomeBloc(),
+      child: MaterialApp(
+        debugShowCheckedModeBanner: false,
+        title: 'Generate',
+        theme: ThemeData(
+          primarySwatch: Colors.teal,
+        ),
+        home: const HomePage(title: 'Generate numbers'),
+      ),
+    );
+  }
+}
+
+class HomePage extends StatefulWidget {
+  const HomePage({Key? key, required this.title}) : super(key: key);
+
+  final String title;
+
+  @override
+  State<HomePage> createState() => _HomePageState();
+}
+
+class _HomePageState extends State<HomePage> {
+  late TextEditingController _controller;
+
```

Comment:
Using of a stateful widget is redundant here, since we are using a statemanagement bloc

Reply:
In my opinion I need to use stateful widget here for controller

Reply:
agree with egor

Reply:
@sergei-mikhailovskii-idf  do I actually need to do it?

Reply:
if only for controller, I think it's ok to define variable TextEditingController _controller = TextEditingController(); like this, and make stateless widget. but if u need it for some other purposes - let's discuss

Reply:
I need to dispose controller and I can do it in Statefull w. only.